### DISCUSSION TITLE
Add the ability to set 'Require a password reset on next login' when importing users

### DISCUSSION THREAD

**Discussion by terryupton** [engram-design/FeedMe#1017] [Ideas]
### Description
Basically as per the thread subject. It would be great if we could force tick this when importing users to the system.

I am migrating users across from an old NON-Craft CMS, as part of this process, we are going to ensure all users have new/better passwords by making them reset their password upon login to the new website/members area.

It would have been handy to have being able to check this when I did the data import.

**Comment by splendidrob** [comment]
Just came across exactly the same issue. It stumped me for a while as I was hoping to get all new users to go through the "forgot password" flow to create a new password but this results in an "invalid email" (Confusing!!) error for any users that have been imported without a password (I'm assuming this is fairly common as it doesnt seem like exporting users passwords is normally an option). I got round this by generating a random password for each user in the csv file before the import but it would be ideal to force the creation of a new password for imported users as an option!
